services/gundeck/test/unit/ThreadBudget.hs

```diff
@@ -286,9 +287,7 @@ mock (Model Nothing) (Init _) =
   InitResponse <$> genSym
 mock (Model (Just _)) Run {} = pure RunResponse
 mock (Model (Just _)) Wait {} = pure WaitResponse
-mock (Model (Just _)) Measure {} = pure MeasureResponse {..}
-  where
-    rspConcreteRunning = undefined
+mock (Model (Just _)) Measure {} = pure (MeasureResponse undefined)
```

Comment:
ok, this has nothing to do with warnings any more, but it's definitely an improvement.  thanks!  :)

Reply:
Well, it followed from the warnings. I removed the `rspConcreteRunning` field label, because it's a partial accessor (`-Wpartial-fields` complained). Without it, I was not able to use the `{..}` syntax anymore.